src/common/i18n/translations/en.json

```diff
@@ -0,0 +1,362 @@
+{
+  "loginScanCard": {
+    "loginWithQR": "Please log in with your unique QR code provided by your supervisor",
+    "scanToLogin": "Scan to login",
+    "needHelp": "Need help?",
+    "askQuestion": "Ask a question"
+  },
+  "loginMobileNumberCard": {
+    "enterMobileNumber": "Please enter your mobile phone number to receive a one-time password.",
+    "mobileNumber": "Mobile phone number",
+    "sendOtp": "Send OTP"
+  },
+  "loginOTPCard": {
+    "sendingOtp": "We're sending you the one-time password...",
+    "Otp": "OTP",
+    "resendIn": "Resend in %{ss}s",
+    "submit": "Submit",
+    "resend": "Resend"
+  },
+  "collectCustomerDetailsScreen": {
+    "checkEligibleItems": "Check the number of item(s) eligible for redemption",
+    "scanIdentification": "Scan identification",
+    "or": "Or",
+    "enterIdNumber": "Enter ID Number",
+    "check": "Check"
+  },
+  "idScanner": {
+    "enterIdManually": "Enter ID manually",
+    "enterManually": "Enter manually",
+    "enterVoucherId": "Enter voucher ID",
+    "scanToCheck": "Scan to check validity"
+  },
+  "navigationDrawer": {
+    "logout": "Logout",
+    "addCampaign": "Add Campaign",
+    "selectCampaign": "Select campaign",
+    "changeChampaign": "Change Campaign",
+    "helpSupport": "Help & Support",
+    "termsOfUse": "Terms of use",
+    "privacyStatement": "Privacy Statement",
+    "reportVulnerability": "Report Vulnerability"
+  },
+  "customerQuotaScreen": {
+    "campaignExpiredOn": "Expired on %{dateTime}",
+    "campaignValidTo": "Valid till %{dateTime}",
+    "quotaRedeemedLimitReachedIDNumber": "Identification number(s)",
+    "quotaCategoryMealCredits": "Meal Credit(s)",
+    "quotaLimitMax": "Max",
+    "quotaAppealCancel": "Cancel",
+    "quotaButtonCheckout": "Checkout",
+    "quotaButtonAdd": "+ Add",
+    "quotaButtonAddVoucher": "Add voucher",
+    "quotaAddId": "Add another ID to combine customer quotas",
+    "quotaCategoryCommunityFoodPack": "Community Food Pack",
+    "quotaCategoryBread": "Bread",
+    "quotaCategoryEggs": "Eggs",
+    "quotaCategoryFruits": "Fruits",
+    "quotaCategoryRootVegetables": "Vegetables",
+    "quotaCategoryFreshVegetables": "Fresh Vegetables",
+    "quotaCategoryChicken": "Chicken (Fresh/frozen/processed)",
+    "quotaCategoryCannedFood": "Canned Food (Meat/seafood/vegetables)",
+    "quotaCategoryPaperProducts": "Paper Products",
+    "quotaCategoryInstantNoodles": "Instant Noodles",
+    "quotaCategoryRice": "Rice",
+    "quotaCategoryCookingOil": "Cooking Oil",
+    "quotaCategoryTTToken": "TT Token",
+    "quotaCategoryVouchers": "Voucher(s):",
+    "quotaCategoryAppreciationGift": "Appreciation Gift",
+    "quotaIdentifierDeviceCode": "Device code",
+    "quotaIdentifierContactNumber": "Contact number",
+    "quotaIdentifierFirstVoucherCode": "First voucher code",
+    "quotaIdentifierButtonScan": "Scan",
```

Comment:
Hmm aren't these dynamic copies? Why are they here?

Reply:
Hmm I think because that time we just shifted all the translated strings from the gdocs into the JSON but we can remove them!

Reply:
Yup let's do since they won't be kept in these json files when we implement for dynamic copies!